src/transactions/test/TxEnvelopeTests.cpp

```diff
@@ -2031,6 +2031,9 @@ TEST_CASE_VERSIONS("txenvelope", "[tx][envelope]")
 
                     REQUIRE(tx1->getResultCode() == txSUCCESS);
                     REQUIRE(tx2->getResultCode() == txFAILED);
+                    OperationResult res = getFirstResult(*tx2);
```

Comment:
This seems like some leftover change, is this intentional?

Reply:
Yes, this was leftover from some debug printing statement, removed.